src/message-store-sql.ts

```diff
@@ -187,20 +234,25 @@ export class MessageStoreSql implements MessageStore {
 
     options?.signal?.throwIfAborted();
 
+    // extract sort property and direction from the supplied messageSort
+    const { property: sortProperty, direction: sortDirection } = this.extractSortProperties(messageSort);
+
     let query = this.#db
       .selectFrom('messageStore')
```

Comment:
You mean for the table names? It can be done, and I'm not necessarily opposed to it (it's just a couple of lines).
However, this would force Google to re-create their DB, which may be ok, but just have to keep that in mind.

Otherwise everything else in this PR gracefully upgrades since it's new tables.

Moving forward I will need to tackle a migration strategy for any breaking changes or new columns. Kylsey offers migration capabilities, but just gotta think about how we want to implement them.

Reply:
Let's do it, I wouldn't worry about Google right now as they'll need to deploy/redeploy regardless. Only reason is that when I see `messageStore` I think DB name, not table name. But only if no battle is involved!

Reply:
Done!

I also updated `eventLogWatermark` to `eventWatermark` in the reference table because the constraint name in `mySQL` was 1 char longer than the allowed.